models/todo.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+
+  const Model = sequelize.Sequelize.Model
+
+  class Todo extends Model {
+    static associate (models) {
+      Todo.belongsTo(models.Status)
+      Todo.belongsTo(models.User)
+    }
+  }
+
+  Todo.init({
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notNull : {
+          msg : 'Title is required'
+        },
+        notEmpty: {
+          args: true,
+          msg : 'Title is required'
+        }
+      }
+    }, 
+    description: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notNull : {
+          msg : 'description is required'
+        },
+        notEmpty: {
+          args: true,
+          msg : 'description is required'
+        }
+      }
+    }, 
+    due_date: {
+      type: DataTypes.DATE,
+      allowNull: false,
+      validate: {
+        notNull : {
+          msg : 'duedate is required'
+        },
+        notEmpty: {
+          args: true,
+          msg : 'duedate is required'
+        },
+        isAfter: {
+          args : `${new Date().toLocaleDateString()}`,
+          msg : 'Date is expired'
```

Comment:
msgnya kalau bisa dibuat jadi lebih jelas. datenya bukan expired, tapi ex: you cannot created data with date that less than current date